Gems/ROS2/Code/Include/ROS2/VehicleDynamics/DriveModels/PidConfiguration.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Serialization/SerializeContext.h>
+#include <control_toolbox/pid.hpp>
+
+namespace VehicleDynamics
```

Comment:
Should this be wrapped in the ROS2 namespace?

Reply:
@michalpelka will introduce the namespace to all VehicleDynamics classes.
Please keep the VehicleDynamics namespace, but add ROS2 namespace as a parent.

Reply:
Done by @michalpelka